includes/specials/SpecialUserlogout.php

```diff
@@ -44,12 +44,6 @@ function execute( $par ) {
 			throw new HttpError( 400, wfMessage( 'suspicious-userlogout' ), wfMessage( 'loginerror' ) );
 		}
 
-		// Some communities (notably uncyclopedia) do not have FacebookClient enabled
-		if ( F::app()->wg->EnableFacebookClientExt ) {
```

Comment:
@mklucsarits because I moved the code to a ```FacebookClient``` file that will only be included if ```EnableFacebookClientExt``` is true.  That's one of the benefits of the hook system (as much as I hate it) which is conditionally included extensions can register for hooks and that code will only be called if the extension has been loaded and you don't have to do extra checks for these ```wg``` variables.